sqldelight-gradle-plugin/src/test/integration-mysql/src/main/sqldelight/com/squareup/sqldelight/mysql/integration/Dog.sq

```diff
@@ -10,4 +10,9 @@ VALUES (?, ?, ?);
 
 selectDogs:
 SELECT *
-FROM dog;
\ No newline at end of file
+FROM dog;
+
+selectDogsWithName:
```

Comment:
we probably want a test that runs this method

Reply:
similar to `selectDogs`, maybe for each dialect as well (postgres/h2)

Reply:
Yeah agree, I had trouble getting the mysql tests running and accidentally committed this, will add it back.

Reply:
you need docker running in case thats why it didnt work

Reply:
Nah it was XCode